.github/workflows/nightly.yml

```diff
@@ -124,3 +124,84 @@ jobs:
           name: darktable.${{ github.sha }}.win64.zip
           path: ${{ env.BUILD_DIR }}/darktable-*.exe
           retention-days: 2
+
+  macOS:
+    if: github.repository == 'darktable-org/darktable' || github.event_name == 'workflow_dispatch'
+    name: Nightly darktable.${{ github.sha }}.macosx
```

Comment:
Perhaps change the "macosx" to something more meaningful/formal like "macOS"? AFACT, "OSX" is no longer to be used?

Reply:
Agreed 👍